### DISCUSSION TITLE
Documentation out of date.

### DISCUSSION THREAD

**Discussion by MACKAT05** [dbt-labs/dbt-snowflake#480] [General]
it looks like there is a disconnect in the documentation.
I see in the code that there is an oauth authentication option but that is missing from:
https://docs.getdbt.com/reference/warehouse-setups/snowflake-setup

**Comment by Victoriapm** [comment]
Hi @MACKAT05, good catch! That same issue has been already opened in the docs repository, you can add your comment and follow the issue [here](https://github.com/dbt-labs/docs.getdbt.com/issues/2892).
